### PR TITLE
Init model for project

### PR DESCRIPTION
**Pull request by hellovietnam93**
Init 9 model in project and relationship between them.

### CODE REVIEW

**Comment by vinhnd2010** [comment]
OK!

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Category < ActiveRecord::Base
+  has_many :lessons, dependent: :destroy
+  has_many :words, dependent: :destroy
+
+  # Validate
```
Comment (dieunb) [inline app/models/category.rb]: remove all remark

File: config/routes.rb [hunk 2]
```diff
@@ -1,63 +1,14 @@
...
-  #     end
-  #   end
-
-  # Example resource route with concerns:
-  #   concern :toggleable do
-  #     post "toggle"
-  #   end
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
+  resources :users do
```
Comment (dieunb) [inline config/routes.rb]: remove

**Comment by vinhnd2010** [comment]
OK!

File: Gemfile [hunk 3]
```diff
@@ -1,30 +1,57 @@
...
+gem "sdoc", "~> 0.4.0", group: :doc
+
+gem "carrierwave",             "0.10.0"
+gem "mini_magick",             "3.8.0"
+gem "fog",                     "1.23.0"
+gem "faker", "1.4.3"
+gem "will_paginate",           "3.0.7"
+gem "bootstrap-will_paginate", "0.0.10"
+gem "bootstrap-sass", "3.2.0.2"
+gem "responders", "~> 2.0"
```
Comment (dieunb) [inline Gemfile]: Remove this Gem

**Pull request merged**